bs-react-native-next/src/apis/ActionSheetIOS.bs.js

```diff
@@ -1,2 +1 @@
-// Generated by BUCKLESCRIPT VERSION 5.0.0, PLEASE EDIT WITH CARE
```

Comment:
I guess that's new? Is this cause the name is .bs.js or does bsb just dropping this "feature"?

Reply:
I added `-bs-no-version-header` to bsconfig.json. ;-)

Reply:
oh I missed that :D